src/lookup-table/types.ts

```diff
@@ -1,14 +1,24 @@
-type LookupTableStructurePropType = {
-  tableName: string
+type LookupTableStructureType = {
+  id?: number
+  name: string
   label?: string
   fieldMap: FieldMapType[]
 }
 
+// Removes 'optional' attributes from a type's properties
+type Concrete<Type> = {
+  [Property in keyof Type]-?: Type[Property]
```

Comment:
What is this for ? I mean what purpose does it server in terms of checking that code is strongly typed ? Why not have two interfaces (one inheriting another ?, one without IDs one with IDs)

Reply:
I wanted to set all the properties of the type required,  and I did not want to create a new type while doing it. 
I just realized that there is a `Required<T>` utility type provided by typescript off the box. I am going to use that and remove the `Concrete<T>` type I created.

Reply:
Resolved with 2f70fb4